### ISSUE TITLE
UI Doesn't display aggregation details if multiple options are present

### ISSUE THREAD

**Issue by keithhand** [kubecost/cost-analyzer-helm-chart#1497]
**Describe the bug**
When selecting multiple aggregation types, if one of the data points is null, the other will not display within the UI. The example below lists the value as `provider` in the UI, even though the API returns `AWS` or `GCP`.

**To Reproduce**
1. [Demo UI link](http://demo.kubecost.io/assets.html?agg=label%3Aapp%2Cprovider&filters=W10%3D)
2. [Demo API link](http://demo.kubecost.io/model/assets?aggregate=label%3Aapp%2Cprovider&filters=W10%3D&window=7d&accumulate=true)

**Expected behavior**
In the UI, if an aggregation type has data available, that data should be shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/43491549/175388321-e49451a6-16c4-49df-be0a-28ae5d664df9.png)

```
{
  code: 200,
    data: [
      {
        __undefined__/ AWS: {
        properties: {
          provider: "AWS",
          service: "Kubernetes",
          cluster: "kc-demo-aws-1"
        },
        labels: {},
        window: {
          start: "2022-06-17T00:00:00Z",
          end: "2022-06-24T00:00:00Z"
        },
        start: "2022-06-17T00:00:00Z",
        end: "2022-06-24T00:00:00Z",
        minutes: 10080,
        adjustment: 0,
        totalCost: 46.891761
      },
      __undefined__ / GCP: {
        properties: {
          provider: "GCP"
        },
        labels: {},
        window: {
          start: "2022-06-17T00:00:00Z",
          end: "2022-06-24T00:00:00Z"
        },
        start: "2022-06-17T00:00:00Z",
        end: "2022-06-24T00:00:00Z",
        minutes: 10080,
        adjustment: 0,
        totalCost: 621.180574
      },
      app = jenkins / GCP: {
        type: "Cloud",
        properties: {
          category: "Compute",
          provider: "GCP",
          account: "01AC9F-74CF1D-5565A2",
          project: "guestbook-227502",
          service: "Compute Engine",
          name: "Storage PD Snapshot in US",
          providerID: "5885458e9c7ea498d7e55af0cd884991"
        },
        labels: {
          app: "jenkins"
        },
        window: {
          start: "2022-06-17T00:00:00Z",
          end: "2022-06-24T00:00:00Z"
        },
        start: "2022-06-17T00:00:00Z",
        end: "2022-06-23T07:00:00Z",
        minutes: 9060,
        adjustment: 0,
        credit: 0,
        totalCost: 0.004683
      }
    }
  ]
}
```

**Comment by dwbrown2** [comment]
@keithhand thanks again for another report! Looks like demo is still running an older version. 

Does this look fixed on latest? http://nightly.kubecost.io:9090/assets.html?agg=service%2Clabel%3Alabel%3Aapp%2Cprovider

**Comment by keithhand** [comment]
@dwbrown2, Yup, I'm still seeing the same results when using my instance on v1.94.3.

**Comment by dwbrown2** [comment]
Strange, I can't repro... any steps you can share? Are you able to recreate via the UI or it just by passing unexpected URL params?

**Comment by keithhand** [comment]
I also see it on the link you posted for the nightly site. The options below show "No service" and "provider" instead of the actual details received from the API.

![image](https://user-images.githubusercontent.com/43491549/175665612-d4cc08a8-fdc4-4757-a680-5987476580ec.png)
[UI](http://nightly.kubecost.io:9090/assets.html?agg=service%2Clabel%3Alabel%3Aapp%2Cprovider)
[API](http://nightly.kubecost.io:9090/model/assets?aggregate=service%2Clabel%3Alabel%3Aapp%2Cprovider&window=7d&accumulate=true)

**Comment by dwbrown2** [comment]
Gotcha, I see it now! Sorry, was looking at this part of the UI from previous screenshot. 

<img width="191" alt="image" src="https://user-images.githubusercontent.com/298359/175666862-75beca8a-43f6-480c-8164-d6d2aca8c070.png">

cc @teevans @AdamStack18 for triage

**Comment by teevans** [comment]
Thanks @keithhand - Assigning to @nealormsbee

**Comment by Adam-Stack-PM** [comment]
@teevans Release blocking or nice to have item?

**Comment by teevans** [comment]
I'd call this blocking. I think it's a big deal when we're not displaying data properly

**Comment by teevans** [comment]
My man.

**Comment by teevans** [comment]
@keithhand - Could I bother you to test a couple scenarios to ensure we closed the loop here?
http://34.139.228.238:9090/

**Comment by keithhand** [comment]
@teevans, Looks good to me. Tested both of the following links, [1](http://34.139.228.238:9090/assets?agg=label%3Aapp%2Cprovider&filters=W10%3D) & [2](http://34.139.228.238:9090/assets?agg=service%2Clabel%3Alabel%3Aapp%2Cprovider)
